### PR TITLE
Prepare 1.2.0-alpha1 release

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ please check the latest changes
 
 ## Unreleased
 
+## 1.2.0-alpha1
+
+Released 2021-Jul-23
+
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: We should have an entry for adding `ConsoleMetricExporter` here, right?

**Review by reyang — APPROVED** [review]
LGTM 👍

File: src/OpenTelemetry.Api/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.2.0-alpha1
+
+Released 2021-Jul-23
+
 * Add Metrics support.
 
 * Removes .NET Framework 4.5.2 support. The minimum .NET Framework
```
Comment (reyang) [inline src/OpenTelemetry.Api/CHANGELOG.md:11]: Just noticed that we didn't mention 4.6 here, is this intentional?

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md [hunk 3]
```diff
@@ -9,8 +9,13 @@ please check the latest changes
...
 * Removes .NET Framework 4.5.2 support. The minimum .NET Framework version
   supported is .NET 4.6.1. ([#2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138))
+* Add Metrics support.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/CHANGELOG.md:18]:
```suggestion
* Add Metrics support. ([#2174](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2174))
```

File: src/OpenTelemetry.Api/CHANGELOG.md (hunk 2 above)
Comment (cijothomas) [reply src/OpenTelemetry.Api/CHANGELOG.md:11]: i think @utpilla commented same in another PR. I initially thought the 2nd sentence was sufficient, but now I think its best to mention removed net452,net46 support :)

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md (hunk 1 above)
Comment (utpilla) [reply src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: Same for OTLP exporter.

File: src/OpenTelemetry.Api/CHANGELOG.md (hunk 2 above)
Comment (reyang) [reply src/OpenTelemetry.Api/CHANGELOG.md:11]: 😸

**Pull request merged**